services/elastic/.devcontainer/Dockerfile

```diff
@@ -0,0 +1,28 @@
+# See here for image contents: https://github.com/microsoft/vscode-dev-containers/tree/v0.195.0/containers/java/.devcontainer/base.Dockerfile
```

Comment:
New Elastic KafkaConsumer service which pushes content into Elasticsearch.  I'd love to replace this with the community free opensource Kafka Connector Elasticsearch Sink.  However, I don't presently have time to figure out how to get it all working.